tests/fault_tolerance/migration/test_vllm.py

```
@@ -198,6 +204,12 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         except Exception as e:
             logging.warning(f"Failed to release vLLM worker port: {e}")
 
+        try:
+            # fpm_port is always allocated in __init__
+            deallocate_port(self.fpm_port)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)